os_command_injection/timothywarner-org_globomantics-robot-fleet-security/vulnerable-demo.js

```
@@ -0,0 +1,149 @@
+function weakEncrypt(data) {
+    // VULNERABLE: Using deprecated MD5 hash
+    return crypto.createHash('md5').update(data).digest('hex');
+}
+
+// 🚨 Insecure Deserialization
+app.post('/api/robot/config', (req, res) => {
+    const configData = req.body.config;
+
+    // VULNERABLE: Deserializing untrusted data
+    try {
+        const config = eval(`(${configData})`); // Using eval() is dangerous!
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)